src/ossf_scorecard/scorecard.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+def get_scorecard(platform, org, repo):
+    url = f"https://api.securityscorecards.dev/projects/{platform}/{org}/{repo}"
+    response = requests.get(url)
+
+    if response.status_code == 200:
+        return response.json()
+    else:
+        response.raise_for_status()
+
+
+def print_scorecard(platform, org, repo):
+    try:
```

Comment:
Let's not print, or have this function. If we want to print something we can use the log, but seems unnecessary here. We only have to store the data correctly in the models at https://github.com/nexB/ScoreCode/pull/4

Reply:
done